packages/techdocs-cli/src/e2e.test.ts

```diff
@@ -9,28 +9,28 @@ describe("end-to-end", () => {
   });
 
   it("can generate", async () => {
-    jest.setTimeout(10000)
-    const proc = await executeTechDocsCliCommand(
-      ["generate", "--no-docker"],
-      {
-        cwd: "../../",
-        killAfter: 8000
-      }
-    );
+    jest.setTimeout(10000);
+    const proc = await executeTechDocsCliCommand(["generate", "--no-docker"], {
+      cwd: "../../",
+      killAfter: 8000,
+    });
+
+    // eslint-disable-next-line no-console
+    console.log(proc.combinedStdOutErr);
```

Comment:
Have been having problems with what seems like flaky tests. They often work locally, but fail in CI for me. This was an attempt to see the output when they do fail. Coincidentally, adding this made the tests pass (at least the first time). These may need to be re-worked in general.

Reply:
An alternative suggestion: if you re-order the expectations below, you'll get `proc.combinedStdOutErr` printed as part of the test fail, while green builds don't get the console log spam.